debian/consul.upstart

```diff
@@ -8,13 +8,15 @@ respawn
 script
   # Make sure to use all our CPUs, because Consul can block a scheduler thread
   export GOMAXPROCS=`nproc`
+  USER="consul"
+  OPTS="agent -config-dir="/etc/consul.d ${CONSUL_FLAGS}"
 
   # Allow overriding env vars in /etc/default/consul
   if [ -f "/etc/default/consul" ]; then
     . /etc/default/consul
   fi
 
-  exec /usr/bin/consul agent \
```

Comment:
This change makes sense. But is there a reason why the `setuid` and `setgid` standard stanzas are not used? They are supported even in precise, and `start-stop-daemon` seems a bit overkill (since PID management should also be covered by upstart itself).